salesProblem/practice_question/classes/cart.rb

```diff
@@ -0,0 +1,51 @@
+require './receipt'
+require '../modules/round_off'
+class Cart
+  attr_accessor :item_name, :price, :exempt, :imported, :tax
+
+  def initialize(item_name, price, exempt, imported)
+		@item_name = item_name

```

Comment:
`item_name, :price, :exempt, :imported, :tax` these are attributes of `Item` not `Cart`. 

Cart attributes can be like `Total Tax, Total Amount, Items List` etc.